<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistribWidget</class>
 <widget class="QWidget" name="DistribWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSigma1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSigma2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTypeDistrib">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>See</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMu1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMu2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelMean">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStdDev">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>StdDev</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStdDev2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>StdDev2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMean2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean2</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxWeight">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> w=</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
